baseapp/abci_utils.go

```diff
@@ -133,13 +126,10 @@ func ValidateVoteExtensions(
 		sumVP += vote.Validator.Power
 	}
 
-	if totalVP > 0 {
-		percentSubmitted := math.LegacyNewDecFromInt(math.NewInt(sumVP)).Quo(math.LegacyNewDecFromInt(math.NewInt(totalVP)))
-		if percentSubmitted.LT(VoteExtensionThreshold) {
-			return fmt.Errorf("insufficient cumulative voting power received to verify vote extensions; got: %s, expected: >=%s", percentSubmitted, VoteExtensionThreshold)
-		}
+	// If the sum of the voting power has not reached (2/3 + 1) we need to error.
+	if sumVP < (totalVP*2/3)+1 {
```

Comment:
Are we sure this won't cause inconsistencies across validator machine architectures?

Reply:
we are testing